src/Scaffolding/VS.Web.CG.Mvc/Identity/IdentityHelper.cs

```diff
@@ -43,7 +43,13 @@ internal static string GetNamespaceFromTypeName(string dbContext)
 
         internal static string GetClassNameFromTypeName(string dbContext)
         {
-            return dbContext.Split(new char[] { '.' }, StringSplitOptions.RemoveEmptyEntries).Last();
+            var lastIndexOfDot = dbContext.LastIndexOf('.');
+            if (lastIndexOfDot == -1)
+            {
+                return string.Empty;
+            }
+
+            return dbContext.Substring(lastIndexOfDot + 1, dbContext.Length - lastIndexOfDot - 1);
```

Comment:
You should just be able to use the [Substring overload that just takes `startIndex`](https://learn.microsoft.com/en-us/dotnet/api/system.string.substring?view=net-8.0#system-string-substring(system-int32)). That'll return everything after that.

Reply:
fixed tyty